server/komodo-rest/src/main/java/org/komodo/rest/service/KomodoTeiidService.java

```diff
@@ -2142,4 +2370,38 @@ public Response ping(final @Context HttpHeaders headers,
             return createErrorResponseWithForbidden(mediaTypes, e, RelationalMessages.Error.TEIID_SERVICE_QUERY_ERROR);
         }
     }
-}
+    
+    /*
+     * Loads default translator mappings from resource file
+     */
+    private void loadDefaultTranslatorMap() {
+        InputStream inputStream = getClass().getClassLoader().getResourceAsStream( DEFAULT_TRANSLATOR_MAPPING_FILE );
+        
+        driverTranslatorMap.clear();
+        
+        // Load the mappings file
+        Document doc;
+        try {
+            String mappingXml = FileUtils.streamToString(inputStream);
+            doc = FileUtils.createDocument(mappingXml);
+        } catch (Exception ex) {
+            LOGGER.error(RelationalMessages.getString(RelationalMessages.Error.TEIID_SERVICE_LOAD_DEFAULT_TRANSLATOR_MAPPINGS_ERROR, ex.getLocalizedMessage()));
+            return;
```

Comment:
Would it be prudent for the exception to be thrown and tossed back to the client with error message? Is the lack of mappings (cos something has gone wrong) worthy of letting the user know? Not sure on this one.